src/model/trust.model.ts

```diff
@@ -9,15 +10,23 @@ export const TrustKey = "trusts";
   be able to do the mapping correctly
 */
 export const TrustKeys: string[] = [
-  "trusts",
+  'trust_id',
+  'trust_name',
+  'creation_date_day',
+  'creation_date_month',
+  'creation_date_year',
+  'unable_to_obtain_all_trust_info',
+  'INDIVIDUALS',
+  'HISTORICAL_BO',
+  'CORPORATES',
 ];
```

Comment:
What does this change do?

Reply:
Following the coding standards, noticed that the keys are added as an array of strings. This did not follow the standard and being a file I was working on I had to correct it. Makes no difference as it is declared but never used can also delete it.

Reply:
Please rollback this change.

The keys are used for marshalling data to/from web forms (except for Trust web pages) using these keys and some functions. This also applies for the form that has the Trust spreadsheet  which is processed in `src/controllers/trust.information.controller.ts` but not to the web forms that we are developing in the Trust Screens development.